ministryofjustice/laa-review-criminal-legal-aid
# [CRIMAP-624] Add a copy button to URN (if present)

## Description of change
Added a copy button to URN (if present) 

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-624

## Notes for reviewer
As discussed with Design team Louise, Olivia and Rosie, I have made the following changes to the URN.

- Added the copy link to say Copy URN
- Changed the text on the left heading to say Unique reference number (URN)
- Removed the Copy URN link from historic applications

## Screenshots of changes (if applicable)

### Before changes:
![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/106679087/6a4f4b62-18f5-4b2f-a2f1-7c2f9227a0d1)

### After changes:
<img width="1297" alt="Screenshot 2023-10-19 at 13 32 46" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/106679087/e6ce5585-84bd-4bad-9f4a-3470ed135bde">

![image](https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/106679087/7db00ddf-ecf5-4e23-8667-39c9b0b52edf)

## How to manually test the feature
- User logs into application
- User clicks on applicants name in the table as shown above in the screenshot in either Your list, All open applications or Closed applications pages.
- User is guided to the applicants page
- User clicks on the copy urn to copy the Unique reference number
- User can then paste the Unique reference number by using ctrl+v onto another application or window to verify the feature is working
